app/components/users/avatar_component.rb

```diff
@@ -32,7 +32,7 @@ class AvatarComponent < ApplicationComponent
     include AvatarHelper
     include OpPrimer::ComponentHelpers
 
-    def initialize(user:, show_name: true, link: true, size: 'default', classes: '', title: nil)
+    def initialize(user:, show_name: true, link: true, size: 'default', classes: '', title: nil, classes_for_name: '')
```

Comment:
I am not super happy with that name. My suggestion would be `name_classes`, which is not really better tbh 🙈  but at least it is consistent to how we name this object in other classes 🤷‍♀️